Includes/ftpConnection.cpp

```diff
@@ -495,7 +510,7 @@ bool ftpConnection::sendFile(std::string const& fileName) {
     outputLine("File opening failed. LOL.\n");
     return false;
   }
-  int bytesToRead = FTP_BUFFER_SIZE - 3;
+  int bytesToRead = FTP_DATA_BUFFER_SIZE - 3;
```

Comment:
I don't understand where the -3 comes from 🤔 not part of the PR tho

Reply:
It was added based on a bad assumption on my part. Could safely be removed. Thanks!